<launch>

   
    <arg name="K_v" default="2" />
    <arg name="K_int" default="0.05" />
    <arg name="K_omega" default="1.5" />
    <arg name="delta_tracking" default="1.5" />
 

    <node name="turtle_traking_control" 
        pkg="turtle_infnet" 
        type="turtle_tracking_control.py"
        args="$(arg K_v) $(arg K_int)  $(arg K_omega)
        		$(arg delta_tracking)"
        output="screen">

	<remap from="master_pose" to="/turtle1/pose"/>
	<remap from="donkey_pose" to="/turtle_donkey/pose"/>
	<remap from="cmd_vel" to="/turtle_donkey/cmd_vel"/>

	</node>
	
</launch>
